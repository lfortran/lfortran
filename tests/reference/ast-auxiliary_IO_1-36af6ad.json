{
    "basename": "ast-auxiliary_IO_1-36af6ad",
    "cmd": "lfortran --fixed-form --show-ast --no-color {infile} -o {outfile}",
    "infile": "tests/fixed_form/auxiliary_IO_1.f",
    "infile_hash": "6be10531c17bcd0a753784768cbd756393fcb08d05bae273201dbe30",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "ast-auxiliary_IO_1-36af6ad.stdout",
    "stdout_hash": "11976ffa040614bba3d9fa046092e4b2a24f52a71271a9ae54b7f03d",
    "stderr": null,
    "stderr_hash": null,
    "returncode": 0
}