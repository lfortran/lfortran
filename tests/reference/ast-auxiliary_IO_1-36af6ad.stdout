(TranslationUnit
    [(Subroutine
        xub
        [(nunit)]
        []
        ()
        ()
        []
        []
        []
        [(Declaration
            (AttrType
                TypeInteger
                []
                ()
                ()
                None
            )
            []
            [(nunit
            []
            []
            ()
            ()
            None
            ())]
            ()
        )
        (Declaration
            (AttrType
                TypeInteger
                []
                ()
                ()
                None
            )
            []
            [(i
            []
            []
            ()
            ()
            None
            ())]
            ()
        )]
        [(DoLoop
            0
            ()
            0
            i
            1
            5
            ()
            [(Backspace
                0
                [nunit]
                []
                ()
            )
            (Rewind
                0
                [nunit]
                []
                ()
            )]
            ()
            ()
        )]
        []
        []
    )
    (Program
        main
        ()
        []
        []
        []
        []
        []
    )]
)
